ansible/config_sonic_basedon_testbed.yml

```diff
@@ -201,7 +201,16 @@
   - name: enable tunnel_qos_remap for T1 in dualtor deployment
     set_fact:
       enable_tunnel_qos_remap: true
-    when: "('leafrouter' == (vm_topo_config['dut_type'] | lower)) and (hwsku in hwsku_list_dualtor_t1) and not (is_ixia_testbed)"
+    when: "((vm_topo_config['dut_type'] | lower) in ['leafrouter', 'backendleafrouter']) and (hwsku in hwsku_list_dualtor_t1) and not (is_ixia_testbed)"
```

Comment:
why do we need to change this line?

Reply:
when dut type is "BackEndLeafRouter" , it also need to support "enable_tunnel_qos_remap".
so I did this change.
and I change to another more readable coding style now, for better understand.